### ISSUE TITLE
SCDoc: more obvious indicator of extension help files

### ISSUE THREAD

**Issue by snappizz** [supercollider/supercollider#2446]
I often find myself checking if something belongs to an extension or not by peeking at its help file. The "(extension)" label at the top of the file alone is easy to miss -- I would also like a little icon or something next to the help file title so it's more visible.

**Comment by telephon** [comment]
Actually, if the extension is in a Quark, it would be good to use or add that `(quark name)` / `(extension from quark name)`?

**Comment by snappizz** [comment]
Sounds good, but we don't have any mechanism in SCDoc for determining the name of the extension, do we?

**Comment by telephon** [comment]
`filenameSymbol`?

like: `Array.findRespondingMethodFor(\flop).filenameSymbol`

**Comment by snappizz** [comment]
```
Matrix.filenameSymbol
-> /home/nathan/.local/share/SuperCollider/downloaded-quarks/MathLib/classes/various/Matrix.sc
```

Is there any automated way to tell that `MathLib` is the quark name?

**Comment by telephon** [comment]
maybe match with those?
```
Quarks.folder;
Quarks.additionalFolders;
```

**Comment by snappizz** [comment]
Works for quarks, not so much for other non-quark extensions like sc3-plugins.

We could introduce a new SCDoc field that provides the extension name. If the extension name can't be divined, the indicator can just default to "Extension"

**Comment by telephon** [comment]
For me, `Quarks.additionalFolders;` returns also the path for sc3-plugins. But maybe it should rather be `LanguageConfig.includePaths`.

**Comment by snappizz** [comment]
Resolved in #2449.
